matisse/src/main/java/com/zhihu/matisse/internal/model/AlbumMediaCollection.java

```diff
@@ -51,7 +51,7 @@
         }
 
         return AlbumMediaLoader.newInstance(context, album,
-                album.isAll() && args.getBoolean(ARGS_ENABLE_CAPTURE, false));
+                (AlbumMediaLoader.Capture) args.getSerializable(ARGS_CAPTURE));
```

Comment:
I notice that you removed `album.isAll()` condition. This will make capture entries appear on every album page.